infrastructure/cdn-in-a-box/traffic_ops/adduser.pl

```diff
@@ -24,23 +24,27 @@
 use Crypt::ScryptKDF qw{ scrypt_hash };
 
 if ($#ARGV < 2) {
-    die "Usage: $ARGV[0] <username> <password> <role>\n";
+    die "Usage: $ARGV[0] USERNAME PASSWORD [ROLE] [TENANT] [EMAIL] [FULL_NAME]\n";
 }
 
 my $username = shift // 'admin';
 my $password = shift or die "Password is required\n";
 my $role = shift // 'admin';
 my $tenant = shift // 'root';
+my $email = shift // 'admin@no-reply.trafficops.infra.ciab.test';
```

Comment:
Might as well set `TO_EMAIL` to this value and use that, since `TO_EMAIL` is not currently used anywhere. Perl can be used to retrieve it.